src/Microsoft.AspNetCore.Mvc.Core/Internal/FileContentResultExecutor.cs

```diff
@@ -15,15 +17,54 @@ public FileContentResultExecutor(ILoggerFactory loggerFactory)
 
         public Task ExecuteAsync(ActionContext context, FileContentResult result)
         {
-            SetHeadersAndLog(context, result);
-            return WriteFileAsync(context, result);
+            RangeItemHeaderValue range;
+            long rangeLength;
+            if (result.LastModified.HasValue)
```

Comment:
This check is to ensure LastModified.Value does not throw. But both params are optional and have a default null value.